lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Wallets.hs

```diff
@@ -462,18 +464,23 @@ spec = do
                   )
                 , ( show addrPoolMax
                   , addrPoolMax
-                  , [ expectResponseCode @IO HTTP.status201 ]
+                  , [ expectResponseCode @IO HTTP.status201
+                    , expectField (#addressPoolGap . #getApiT) (`shouldBe` maxBound)
+                  ]
                   )
                 ]
         forM_ matrix $ \(title, addrPoolGap, expectations) -> it title $ \ctx -> do
-            let payload = Json [json| {
-                    "name": "Secure Wallet",
-                    "mnemonic_sentence": #{mnemonics24},
-                    "passphrase": "Secure passphrase",
-                    "address_pool_gap": #{addrPoolGap}
-                    } |]
-            r <- request @ApiWallet ctx (Link.postWallet @'Shelley) Default payload
-            verify r expectations
+            let payload = payloadWith' "Secure Wallet" mnemonics24 (fromIntegral addrPoolGap)
+            rW <- request @ApiWallet ctx (Link.postWallet @'Shelley) Default payload
+            verify rW expectations
+            let w = getFromResponse id rW
+            rA <- request @[ApiAddress n] ctx
+                (Link.listAddresses @'Shelley w) Default Empty
+            _ <- request @ApiWallet ctx
+                (Link.deleteWallet @'Shelley w) Default Empty
```

Comment:
Test passes for me locally as well... Could this line be somehow problematic? I guess we don't need to remove the wallet (Just thinking out loud :man_shrugging: )

Reply:
oh, ok, haven't noticed that it uses hard-coded mnemonic...